src/Chapter2.hs

```diff
@@ -621,7 +653,14 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+
+takeEven :: [a] -> [a]
+takeEven = go True
+  where
+  go _ [] = []
+  go pos (x:xs)
+    | pos = x : go (not pos) xs
+    | otherwise = go (not pos) xs
```

Comment:
Nice idea on the helper functions :+1:

Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂